app/js/handlebarsScoreTable.ts

```diff
@@ -328,13 +331,14 @@ async function addEventListenersForViewResults() {
                         if (user.id === resultData.data.id) {
                             let testId = user.test_id
                             getData("question?test_id=" + testId).then(questionData => {
-                                resultsTable.innerHTML = template(createUserResults(resultData, questionData));
+                                const html = template(createUserResults(resultData, questionData));
+                                resultsTable.innerHTML = html
+                                console.log(html)
```

Comment:
u left in my debug code, please put back to how it was before, remove the console.log and the unnecessary variable